GETTING_STARTED.md

```diff
@@ -0,0 +1,63 @@
+# Getting started with Beluga
+
+## Crash course
+
+1. **Clone the repository**. You will need `git`.
+   ```bash
+   git clone --recursive git@github.com:Ekumen-OS/beluga.git
+   ```
+
+1. **Build and run the development docker container**. You will need [`docker-compose v2.10+`](https://github.com/docker/compose/tree/v2).
```

Comment:
All items seem to be numbered "1". Is that right?

Reply:
That's right, if we ever want to add a step, the markdown rendering will take care of the numbering.